keyboards/ymd96/info.json

```diff
@@ -0,0 +1,16 @@
+{
+    "keyboard_name": "ymd96", 
+    "url": "", 
+    "maintainer": "Andrew", 
+    "bootloader": "bootloaderHID", 
+    "width": 19, 
+    "height": 6, 
+    "layouts": {
+        "LAYOUT": {
```

Comment:
This layout macro name `LAYOUT` does not match any of the existing layout macros in ymd96.h.